Libraries/Components/TextInput/TextInput.js

```diff
@@ -270,6 +270,10 @@ type IOSProps = $ReadOnly<{|
   PasswordRules?: ?PasswordRules,
 
   /*
+   * If `true`, allows TextInput to pass touch events to the parent component.
+   * This allows components such as SwipeableListView to be swipeable from the TextInput on iOS,
```

Comment:
SwipeableListView doesn't exist anymore

Reply:
@TheSavior 
Thanks for the comment! I've removed SwipeableListView from the description.